Mathlib/CategoryTheory/Adjunction/FullyFaithful.lean

```diff
@@ -178,6 +178,30 @@ instance whiskerRight_unit_iso_of_R_fully_faithful [R.Full] [R.Faithful] :
 set_option linter.uppercaseLean3 false in
 #align category_theory.whisker_right_unit_iso_of_R_fully_faithful CategoryTheory.whiskerRight_unit_iso_of_R_fully_faithful
 
+instance [L.Faithful] [L.Full] {Y : C} : IsIso (h.counit.app (L.obj Y)) :=
+  isIso_of_hom_comp_eq_id _ (h.left_triangle_components Y)
+
+lemma isIso_counit_app_iff_mem_essImage [L.Faithful] [L.Full] (X : D) :
```

Comment:
```suggestion
lemma isIso_counit_app_iff_mem_essImage [L.Faithful] [L.Full] {X : D} :
```
Also you can maybe specialize this into the old lemma if you think this would make it easier to use?